core/src/autogluon/core/features/feature_metadata.py

```diff
@@ -176,6 +177,17 @@ def keep_features(self, features: list, inplace=False):
         features_to_remove = [feature for feature in self.get_features() if feature not in features]
         return self.remove_features(features=features_to_remove, inplace=inplace)
 
+    def add_special_types(self, type_map_special: dict, inplace=False):
+        """Adds special types to features"""
```

Comment:
might be helpful if docstring provides a basic example use-case.

Reply:
added